### ISSUE TITLE
False error inspection for E0277 about Fail trait

### ISSUE THREAD

**Issue by surfingtomchen** [intellij-rust/intellij-rust#6101]
## Environment

* **IntelliJ Rust plugin version:** 0.3.130.3337-202
* **Rust toolchain version:** 1.44.1
* **IDE name and version:** CLION, 2020.2.1
* **Operating system:** MacOS 10.15.6

## Problem description

The compile succeed but the IDE shows error about E0277. Actually if implement the ```Fail``` trait, ```Display``` trait should not be implemented.

## Steps to reproduce
here is the snapshot of the code:

<img width="685" alt="Screen Shot 2020-09-11 at 5 51 21 PM" src="https://user-images.githubusercontent.com/1343798/92983314-745ff800-f457-11ea-8912-2c428b6ca350.png">

**Comment by Undin** [comment]
@surfingtomchen Thanks foe the report!
Unfortunately, proc macros are not supported yet
See #1786 for more details

**Comment by Undin** [comment]
Duplicate of #5954
